tools/setup/install-aws-server

```diff
@@ -97,7 +98,7 @@ passwd -d root
 apt-get update
 apt-get -y upgrade
 
-cd "$(mktemp -d)"
+cd "\$(mktemp -d)"
```

Comment:
This was previously creating a temporary directory on the client and `cd`ing to it on the server.